src/plugins/safeSnap/utils/umaModule.ts

```diff
@@ -215,20 +198,26 @@ export const getModuleDetailsUma = async (
   // Get the full proposal events (with state).
   const fullAssertionEvent = await Promise.all(
     thisModuleAssertionEvent.map(async event => {
-      return oracleContract
-        .getAssertion(event.args?.assertionId)
-        .then(result => {
-          const isExpired =
-            Math.floor(Date.now() / 1000) >= Number(result.expirationTime);
+      const assertion = oracleContract.getAssertion(
+        event.args?.assertionId
+      ) as Promise<{
+        expirationTime: BigNumber;
+        settled: boolean;
+      }>;
+
+      return assertion.then(result => {
+        const isExpired =
+          Math.floor(Date.now() / 1000) >= Number(result.expirationTime);
 
-          return {
-            expirationTimestamp: result.expirationTime,
-            isExpired: isExpired,
-            isSettled: result.settled,
-            proposalHash: proposalHash,
-            proposalTxHash: event.transactionHash
-          };
-        });
+        return {
+          expirationTimestamp: result.expirationTime,
+          isExpired: isExpired,
+          isSettled: result.settled,
+          proposalHash: proposalHash,
+          proposalTxHash: event.transactionHash,
+          logIndex: event.logIndex
```

Comment:
added logIndex which is expected in the `getProposalUrl` function in `HandleOutcomeUma`